kview/src/main/java/dev/ikm/komet/kview/klfields/KlFieldHelper.java

```diff
@@ -83,6 +85,9 @@ public static Node generateNode(FieldRecord fieldRecord, ObservableField observa
         } else if (dataTypeNid == TinkarTerm.BOOLEAN_FIELD.nid()) {
             KlBooleanFieldFactory klBooleanFieldFactory = new KlBooleanFieldFactory();
             node = klBooleanFieldFactory.create(observableField, viewProperties.nodeView(), editable).klWidget();
+        } else if (dataTypeNid == TinkarTerm.IMAGE_FIELD.nid()) {
+            KlImageFieldFactory imageFieldFactory = new KlImageFieldFactory();
+            node = imageFieldFactory.create(observableField, viewProperties.nodeView(), editable).klWidget();
```

Comment:
Not sure why GitHub doesn't allow me to add feedback to code lines below, but I have a question.
For Byte array field why is this code returning an image custom control?
Maybe this is older code.